tests/acceptance/09_services/custom.cf

```diff
@@ -0,0 +1,61 @@
+#######################################################
+#
+# Test services custom keyword
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "dummy" string => "dummy";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      "test_suppress_fail" string => "windows",
+        meta => { "redmine4772" };
+
+  services:
+      "myservice"
+      service_policy => "custom",
```

Comment:
An acceptance test for the default value would be nice to have, i.e. omit completely this line, and do not set `service_policy` at all.